csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -291,6 +291,47 @@ private void CanRunInferenceOnAModel(GraphOptimizationLevel graphOptimizationLev
             }
         }
 
+        [Fact]
+        public void InferenceSessionManualDispose()
+        {
+            string modelPath = Path.Combine(Directory.GetCurrentDirectory(), "squeezenet.onnx");
+
+            // Set the graph optimization level for this session.
+            SessionOptions options = new SessionOptions();
+            options.EnableProfiling = false;
```

Comment:
Why does the unit test for a change to test calling EndProfiling disable profiling?

Reply:
I was experimenting and committed this after I marked the PR WIP. I have fixed this.